generator/test_case_generator.go

```diff
@@ -60,35 +70,29 @@ func genMatrix(targetOS string, testDirList []string) []map[string]string {
 	var testMatrix []map[string]string
```

Comment:
I do wonder if these names can be better because I remember being pretty confused when I first worked on this repo, but think it's out of scope of this PR because it's been like that already. If you have ideas & willing, could do it but otherwise non blocking!